### ISSUE TITLE
RDF::Vocab::IANA['first'] no longer works

### ISSUE THREAD

**Issue by jcoyne** [ruby-rdf/rdf-vocab#32]
```
/home/travis/build/projecthydra-labs/curation_concerns/vendor/bundle/ruby/2.2.0/gems/rdf-1.99.0/lib/rdf/vocabulary.rb:739:in `rescue in []': first not found in vocabulary RDF::Vocab::IANA (KeyError)

	from /home/travis/build/projecthydra-labs/curation_concerns/vendor/bundle/ruby/2.2.0/gems/rdf-1.99.0/lib/rdf/vocabulary.rb:737:in `[]'

	from /home/travis/build/projecthydra-labs/curation_concerns/vendor/bundle/ruby/2.2.0/gems/activefedora-aggregation-0.6.0/lib/active_fedora/orders/builder.rb:27:in `block in build'

	from /home/travis/build/projecthydra-labs/curation_concerns/vendor/bundle/ruby/2.2.0/gems/activefedora-aggregation-0.6.0/lib/active_fedora/orders/builder.rb:26:in `tap'

	from /home/travis/build/projecthydra-labs/curation_concerns/vendor/bundle/ruby/2.2.0/gems/activefedora-aggregation-0.6.0/lib/active_fedora/orders/builder.rb:26:in `build'

	from /home/travis/build/projecthydra-labs/curation_concerns/vendor/bundle/ruby/2.2.0/gems/active-fedora-9.6.1/lib/active_fedora/associations/builder/association.rb:13:in `build'

	from /home/travis/build/projecthydra-labs/curation_concerns/vendor/bundle/ruby/2.2.0/gems/activefedora-aggregation-0.6.0/lib/active_fedora/aggregation/base_extension.rb:58:in `orders'
```

**Comment by jcoyne** [comment]
```
irb(main):001:0> RDF::Vocab::IANA['first']
=> #<RDF::Vocabulary::Term:0x3fef0b95c600 URI:http://www.iana.org/assignments/link-relations/first>
irb(main):002:0> exit
activefedora-aggregation [master]$ bundle show rdf-vocab
/Users/justin/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/rdf-vocab-0.8.6
```

Error occurs when upgrading rdf-vocab to 0.8.7
```
RDF::Vocab::IANA['first']
KeyError: first not found in vocabulary RDF::Vocab::IANA
```

**Comment by acoburn** [comment]
I believe this issue is described here: #31

**Comment by jcoyne** [comment]
@acoburn No problem with the URI's changing, but I don't have a way to get the `first` term.

**Comment by gkellogg** [comment]
Sorry, something else must have been messed up; I'll check and fix/release later today. AFK right now

**Comment by jcoyne** [comment]
@gkellogg Ok. Thanks.

**Comment by jcoyne** [comment]
Looks like all the properties went away actually.

**Comment by acoburn** [comment]
@jcoyne yes exactly: https://github.com/ruby-rdf/rdf-vocab/blob/develop/lib/rdf/vocab/iana.rb is now empty

**Comment by grosscol** [comment]
I don't see why the properties file `vocab/iana.rb` wasn't properly generated from the .ttl file.  Is there a url mismatch?

https://github.com/ruby-rdf/rdf-vocab/commit/2314c1302e93c04d2cff4e93d4ceb7d0e591d430

**Comment by grosscol** [comment]
Looks like the vocab-fetch was looking for the wrong url, or at least that's why I assume given the ouput.

`# Vocabulary for <http://www.iana.org/assignments/link-relations/>`

**Comment by gkellogg** [comment]
That's probably it, updates the source file, but not the vocabulary def.

**Comment by gkellogg** [comment]
Fixed in d443106bd8fad69cf51f9af0187d883b84586663 and released in 0.8.7.1.
